paulshryock/paul-shryock
# 32: Sanitize CSS, inline CSS @import rules content

## Summary
Include `sanitize.css` programmatically.

## Testing
- Run `npm start`
- See that (non-purged) CSS sanitization rules are included in output CSS.

## Issue(s)

Closes #32

## Changelog

### Added
- Sanitize CSS. [#32]

### Changed
- Inline CSS @import rules content.

## Release Checklist
